### ISSUE TITLE
Documentation error: Install - Dependencies

### ISSUE THREAD

**Issue by kimjohn1** [ComputationalRadiationPhysics/picongpu#3989]
Maintainers,

New guy here, going through an initial installation.  In Read The Docs (v:latest), Section Install, Dependencies, Requirements, Mandatory, PIConGPU Source Code, the instruction is:
    git clone https://github.com/ComputationalRadiationPhysics/picongpu.git $HOME/src/picongpu

Note the destination is $HOME/src/picongpu

The next instruction, under 'environment', is:
    export PICSRC=$PICHOME/src/picongpu

Note the path is $PICHOME/src/picongpu

There appears to be a discrepancy between the location of the picongpu clone ($HOME/src/picongpu) and the path that is addressing that clone ($PICHOME/src/picongpu).  I didn't see $PICHOME defined earlier in the installation steps, and I believe in that case, bash takes the undefined $PICHOME to $HOME, so there would be no functional error here.  However, as it reads right now and given the emphasis on correctly setting environment variables throughout the installation, it might give someone unfamiliar with PIConGPU, like me, a bit of concern to see this unexplained difference.

Let me know if there is a different forum for an editorial suggestion like this.

Kim

**Comment by kimjohn1** [comment]
Added note: I very much appreciate the 'Your Workstation' section that has recently been added, and it has been quite useful for me.  While using it I found another editorial change that might be good to make; In Read The Docs (v:latest), section Install, picongpu.profile, the newly added Your Workstation, in the text of the Your Workstation profile, near the bottom where the variables and paths are being set, there is a re-definition of PICSRC:
    export PICSRC=$HOME/WORK/src/picongpu

That declaration is unnecessary given that PICSRC is defined in the earlier section of the Install process (described in the original issue statement above).  The five export statements that follow the PICSRC declaration in Your Workstation are duplicates of the steps that follow cloning the PIConGPU Source Code, so they could be commented out here, or noted to only be needed if not previously set.

Kim

**Comment by sbastrakov** [comment]
Hello @kimjohn1 ,

Great catch, `$PICHOME` is a variable we used a long time ago, but no more. So in that documentation piece it indeed should be replaced with `$HOME` as you suggested, I will fix it shortly.

Regarding your second post, I do not think it is a redifinition. In a "daily" workflow, a user starts with sourcing one of the profiles, e.g. for workstation. And does not manually do [these steps](https://picongpu.readthedocs.io/en/latest/install/dependencies.html#picongpu-source-code) every time, those are to be done once when getting started with PIConGPU. We should probably make the distinction more clear. Also, I noticed an inconsistency between  `$PICSRC` in the workstation profile and the rest of them, and the "getting started" one, this could probably be fixed as well.
